app/services/feeds/import.rb

```diff
@@ -71,7 +71,9 @@ def fetch_feeds(batch_of_users)
         next if cleaned_url.blank?
 
         response = ForemStatsClient.time("feeds::import::fetch_feed", tags: ["user_id:#{user_id}", "url:#{url}"]) do
-          HTTParty.get(cleaned_url, timeout: 10)
+          HTTParty.get(cleaned_url,
+                       timeout: 10,
+                       headers: { "User-Agent" => "Forem Feeds Import" })
```

Comment:
Nitpick/naming suggestion: 

```suggestion
                       headers: { "User-Agent" => "Forem Feeds Importer" })
```